package.json

```diff
@@ -72,15 +74,16 @@
     "@types/jest": "^26.0.20",
     "@types/node": "12",
     "@types/request": "^2.48.5",
-    "babel-eslint": "^10.1.0",
+    "@typescript-eslint/eslint-plugin": "^4.17.0",
+    "@typescript-eslint/parser": "^4.17.0",
     "babel-jest": "^26.6.3",
     "babelify": "^10.0.0",
     "better-docs": "^2.3.2",
     "browserify": "^17.0.0",
     "docdash": "^1.2.0",
     "eslint": "7.18.0",
-    "eslint-config-matrix-org": "^0.2.0",
-    "eslint-plugin-babel": "^5.3.1",
+    "eslint-config-google": "^0.14.0",
```

Comment:
hmm, why are we pulling in the google config here?

Reply:
Our configs [extend from it](https://github.com/matrix-org/eslint-plugin-matrix-org/pull/5/files#diff-92ae86c53e137522d28b1e7e7d87ecba8c83f4f71baa1dce5142a1c788fa7d51R7), and since I've [removed all the deps](https://github.com/matrix-org/eslint-plugin-matrix-org/pull/5/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R75-R88) from our own package, it appears here.

Reply:
hmm, interesting